test/e2e/flask/petnames-transactions.spec.js

```diff
@@ -13,13 +14,24 @@ const {
   saveName,
 } = require('./petnames-helpers');
 
-async function createTransactionRequest(driver) {
+async function createTransactionRequestInDapp(driver) {
   await driver.clickElement('#sendButton');
   await driver.delay(3000);
 }
 
-describe('Petnames', function () {
-  it('can save names for addresses in transaction requests', async function () {
+async function createTransactionRequestInWallet(driver, recipientAddress) {
```

Comment:
Minor, but perhaps more explicit to call these `createDappTransaction` and `createSendTransaction`?

Reply:
Both transactions appear to be sending ETH. The key difference seems to be the origin of the request: one comes from the dApp, and the other from the wallet.

I've made changes to the names accordingly. Let me know what you think.